etest/docker/container.py

```diff
@@ -24,27 +24,30 @@ def commit(container: Container, tag: str, repository: str, *args, **kwargs):
     return repo + ":" + tag
 
 
-def create(overlay: str, *args, **kwargs):
+def create(overlay="", autoconf=True, *args, **kwargs):
     """Create a Docker container."""
-    container_data = common.API_CLIENT.create_container(
-        *args,
-        **kwargs,
-        host_config=common.API_CLIENT.create_host_config(
-            binds={
-                overlay: {
-                    "bind": "/overlay",
-                    "ro": True,
+    if autoconf:
```

Comment:
It's not clear to me what autoconf does.  Can we extract configuration somehow and make this a constructor instead?  That might make this more clear.

Reply:
It decides whether to add parameters automatically or not. A new function may have been a better option.

Reply:
I think having a new function would make this much more clear and keep it so you can more easily add tests.